backend/core/domains/contracts/views.py

```
@@ -153,13 +175,20 @@ def get_serializer_class(self):
         return EventContractSerializer
     
     def retrieve(self, request, *args, **kwargs):
```

Ruff (0.13.3)

Unused method argument: `request`

(ARG002)